src/formio/components/Map.jsx

```diff
@@ -115,6 +115,7 @@ export default class Map extends Field {
             onMarkerSet={this.onMarkerSet.bind(this)}
             defaultCenter={defaultCenter}
             defaultZoomLevel={zoom || DEFAULT_ZOOM}
+            tileUrl={this.component.url}
```

Comment:
I think it would be wise to already define the type in the `types` repository for this, when we have the custom renderer we can build on top of that.

Additionally, I think a property `tileLayerUrl` is much better than just `url` - the latter is too generic and will be confusing when you add URLs/configurations for the feature layers.

Reply:
With the types repository you mean https://github.com/open-formulieren/types, right?

Check, yeah it would be best to keep that naming consistent.. I'll change it

Reply:
indeed